rules_python/whl.py

```diff
@@ -63,13 +63,12 @@ def metadata(self):
   def name(self):
     return self.metadata().get('name')
 
-  def dependencies(self):
+  def dependencies(self, extra=None):
```

Comment:
Can you document that passing extra=None means the dependencies of the base wheel?

Reply:
Done